observability/ml-observability.md

```diff
@@ -0,0 +1,44 @@
+# Observability in Machine Learning  
+
+Development process of software system with machine learning component is more complex
+than traditional software as such systems change in three directions: the code, the model and the data.
+We can distinguish 2 stages of such system lifespan: experimentation and production.
```

Comment:
Nit: language details

The two stages of these types of systems, experimentation and production, require different approaches to observability as discussed below: